backend/service/k8s/pods.go

```diff
@@ -62,6 +62,79 @@ func (s *svc) ListPods(ctx context.Context, clientset, cluster, namespace string
 	return pods, nil
 }
 
+func (s *svc) UpdatePodAnnotations(ctx context.Context, clientset, cluster, namespace, name string, expectedAnnotations, newAnnotations Annotations) error {
+	cs, err := s.manager.GetK8sClientset(clientset, cluster, namespace)
+	if err != nil {
+		return err
+	}
+
+	pod, err := cs.CoreV1().Pods(namespace).Get(name, metav1.GetOptions{})
```

Comment:
```suggestion
	pod, err := cs.CoreV1().Pods(cs.Namespace()).Get(name, metav1.GetOptions{})
```

Reply:
`DescribePod` and `ListPods` use `namespace`.  Also `ContextClientset` has no field or method `namespace` =).

Reply:
they also need to be updated.

https://github.com/lyft/clutch/blob/30dc4f3db3cf06b9bb49867c2a8848c7a3ee6f0f/backend/service/k8s/clientset.go#L18

confused where `as no field or method namespace` is coming from based on ^, can you share additional output?

Reply:
Nevermind, I made a typo which was causing the compilation failure =).

I added a commit fixing the other occurrences.